carbonmark/components/ExitModal/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { css } from "@emotion/css";
+
+export const modal = css`
+  .modalContent {
+    width: 37rem !important;
+    .title {
+      & p {
+        font-size: 2.4rem;
+      }
+    }
+  }
+`;
+
+export const content = css`
+  margin-top: 2rem;
+  div {
+    display: flex;
+    font-size: 1.6rem;
+    line-height: 2rem;
+    margin-bottom: 1.2rem;
+
+    & svg {
+      width: 2rem;
+      height: 2rem;
+      fill: #ffb800;
+      margin-top: 0.2rem;
+      margin-right: 1rem;
+    }
+  }
+`;
+
+export const cancelButton = css`
+  width: 100%;
+  margin-top: 1.2rem;
+  border-color: var(--warn);
+  color: var(--warn) !important;
+`;
+
+export const continueButton = css`
+  width: 100%;
+  margin-top: 2.4rem;
+  color: white !important;
```

Comment:
coare you able to explain why !important is neccessary, is there something we can do in how we order classnames for the underlying button components?

Reply:
Let me look into this again and see can if I can resolve without using !important. As far as I remember the underlying colour was black and I couldn't seem to get around it any other way without using !important.